spec/support/redis.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require 'redis'
+
+RSpec.configure do |c|
+  c.before(:suite) do
+    redis_pid = fork { exec 'redis-server spec/support/redis.conf' }
```

Comment:
This is also very similar to how OpenTracing does it. Do you like this approach? Any better idea?

Reply:
Fine for me.